tests/actor/power_distributing/_component_status/test_ev_charger_status.py

```diff
@@ -0,0 +1,165 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Tests for EVChargerStatusTracker."""
+
+import asyncio
+from datetime import datetime, timedelta, timezone
+
+from frequenz.channels import Broadcast
+from frequenz.client.microgrid import EVChargerCableState, EVChargerComponentState
+from pytest_mock import MockerFixture
+
+from frequenz.sdk._internal._asyncio import cancel_and_await
+from frequenz.sdk.actor.power_distributing._component_status import (
+    ComponentStatus,
+    ComponentStatusEnum,
+    EVChargerStatusTracker,
+    SetPowerResult,
+)
+
+from ....timeseries.mock_microgrid import MockMicrogrid
+from ....utils.component_data_wrapper import EvChargerDataWrapper
+from ....utils.receive_timeout import Timeout, receive_timeout
+
+_EV_CHARGER_ID = 6
+
+
+class TestEVChargerStatusTracker:
+    """Tests for EVChargerStatusTracker."""
+
+    async def test_status_changes(self, mocker: MockerFixture) -> None:
```

Comment:
Why are you not using solipsism (and time_machine for fake_time if required)  to prevent real waiting times here, however small they may be?

Reply:
Never figured out how to use it.  Will look into it, but would like to make an issue instead, because I still have to get the PV Pool out in the next couple of weeks.  If that happens quickly, can replace tests for both pools.  Or can do that in May.

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/issues/901